hist/hist/test/test_TFormulaPol.cxx

```diff
@@ -0,0 +1,65 @@
+#include "gtest/gtest.h"
+#include "TFormula.h"
+#include "TF1.h"
+#include "ROOT/TestSupport.hxx"
+#include <cmath>
+
+using namespace ROOT;
+
+class TFormulaPolTest : public ::testing::Test {
+protected:
+    void SetUp() override {
+        TestSupport::CheckDiagsRAII diags;
+    }
+};
+
+TEST(TFormulaPolTest, BasicPolynomialConstruction) {
+    TestSupport::CheckDiagsRAII diags;
```

Comment:
This and the below instances can be removed if the formula doesn't generate any warnings or errors.